### ISSUE TITLE
How do Liverpool civic tech initiatives reach beyond "white male geeks"

### ISSUE THREAD

**Issue by amcewen** [Liverpool-UK/somebody-should#10]
It's glaringly obvious at the moment (given that I've not done much to promote it beyond a few tweets and mentions on the @DoESLiverpool mailing list) that anything reported here is unlikely to be representative of the entire population of (or visitors to) Liverpool.  

It'll be a similar story (at the moment) for http://codeforliverpool.org and https://www.mysociety.org/files/2015/10/demographics-report.pdf shows that it remains a problem for established civic tech projects (although that shows interestingly that in the UK it skews older rather than younger).

How do we reach less tech-savvy (or tech-interested) people?  How do we show them how to take advantage of the opportunities of digital technologies?  Or how do we allow their interest, or lack of it, to inform which initiatives to pursue and which to abandon (because they'll only serve a hyper-geeky niche)?

**Comment by amcewen** [comment]
There was some discussion of this in the http://codeforliverpool.org session at #OggCamp yesterday.  

My thoughts are that...
 * we shouldn't let this issue stop us making a start on building some things that are useful - they'll serve as examples when we're trying to explain it to others and encourage them in.  We need to show that we're a group that *does* things, to build credibility
 * Once we've got some examples under our belt, and a better idea of what the group is doing, then we should be looking to go to where the under-represented communities are and tell them about what we're up to.  Build a slide deck (or something) to let us easily present at Friends of the Earth or Transition Towns conferences; talk to Homebaked Anfield to see if we can hold a session there; reach out to the Granby 4 Streets group; etc. etc.
 * myabe look to building up a team who can provide on-request advice to people with problems to solve.  Something a bit like Social Media Surgeries?  

Although if anyone has better ideas, or better networks with non-tech groups or non-"white male geeks" then by all means get on with fixing things sooner :-D

**Comment by davidtweaver** [comment]
Hmm, I think that asking a question like this on GitHub might already skew the kinds of answers you get - I might repost this elsewhere though, if that's okay with you? I know a fair number of people very interested in civic engagement, from non-tech backgrounds.

**Comment by huffeec** [comment]
I was wondering about this after yesterday. I've been to a few of the
vcsscamp unconferences, it's a really good format for introducing tech
ideas to vol sec orgs.

https://vcsscamp.wordpress.com

Might be that the infrastructure focus isn't quite right for code for
liverpool but something on these lines might be useful? Maybe starting with
a session themed to specific area?

**Comment by amcewen** [comment]
@davidtweaver Please do!  Asking it here was as much about highlighting the deficiency as an issue as (actually probably more) important as any of the others, and act as a place for us to discuss how we address it, as we would with any other issue.

As this discussion shows, there are great benefits to be had in using networked, digital tools - so whilst finding ways (and maybe building tools) to bridge into offline networks is important, I think the aim should be to show people the value in learning the skills to engage here, and help them gain those skills where needed.

@huffeec Totally.  There are sometimes those sorts of things here in Liverpool, aren't there?  My mention of Friends of the Earth and Transition Towns was me doing a bad job of alluding to that sort of thing - for example, we might have been able to run a session at http://www.eventbrite.co.uk/e/make-ends-meet-a-guide-to-sustainable-living-tickets-18006192005 the other week.  And didn't http://www.sentogether.net/ have an unconference a while back?  Or do you mean that we should organise something like VCSSCamp ourselves?

**Comment by huffeec** [comment]
Yes, there are other meetings we can get to, less than there used to be but they're still there. LCVS used to manage a lot of thematic networks like this one (http://www.lcvs.org.uk/havemysay/communities-of-interest/0-25-network--children-and-young-people-.phuse) but there aren't as many post austerity... (I really fancied the transition Liverpool event but I was helping build a knitting machine to a deadline that weekend.)

But I was thinking more of running a specific unconference for geeks and community groups. It would probably end up as part social media surgery, part identifying problems that could use some hacking. Wouldn't take much to set up, would be happy to speak to Paul from VCSS camp for some tips if people thought it'd be useful to have something like this?

**Comment by ka81** [comment]
Hi All , 

Quick question ...
So which non white male Geeks groups are you trying to reach ? Might help to narrow it to start...

**Comment by amcewen** [comment]
@huffeec If you want to run with arranging an unconference that'd be cool.  Sounds like SEN are pondering something similar, so probably worth talking to them about it too.  Feel free to spawn a new issue as a place to gather thoughts/action/etc. around it, if that's useful, as it's a more specific item than the generic "widen our outlook" here (and if you mention "issue #10" in it then github will make a link between the two issues for anyone to find the link :-)

@ka81 Good question.  Not sure I've much of a good answer beyond "all of them", which is exceedingly useless, I know :-D  And you're right, that's more of a long-term aim - in the short term we probably want people who are more civic minded and interested in _doing_ things (although the things to be done don't have to be writing code).  It's a tricky balance in this early stage (I think), as we don't have any examples of what we'd be building - as we get more things built there'll be more to show people to help explain it.  But maybe there are other, better, ways to explain it?  The best I've managed so far is the sort of thing at http://www.mcqn.net/mcfilter/archives/thinking/spreading_the_does_liverpool_ethos.html

In general (and this is an off-the-top-of-my-head list, I'm bound to miss things...) the non-coding tasks would include...
- Design - of all sorts: graphic; user-experience; service...
- Copywriting to improve how things are explained
- Helping get word out about what's built
- Suggesting things that could be built
- Outreach to explain what we're up to to other groups
- Social media surgery sort of activities to help people learn how to use digital tools (inc. basic stuff like Twitter or blogging)

All of which are just as important as the coding, they're just not as immediately obvious (at least to the coders ;-)

For the first meeting or two, the answer is probably "the groups we know and can reach".  We're not going to have masses of coders, so even a few non-geeks should help.  The second meeting is likely to be a cycle hack day (on 23rd January), so cycling-related groups would be an obvious one...

**Comment by ka81** [comment]
Well @amcewen "civic minded and interested in doing" .... I can help with that list and thought maybe as a start I could go visit the WI in Liverpool?
 Had alot to do with the Runcorn branch over the years and they are civic minded. The Liverpool branch looks very cool...
https://m.facebook.com/LiverpoolLiverbirdsWI/

I know the WI are going though a push for younger members http://m.huffpost.com/uk/entry/4934509 and I'm sure that using some of the kit in DoES would be useful to them. Bet they would have some great ideas on improving the community. Maybe ask them to a social ? That's one group....

Caroline

**Comment by ka81** [comment]
P..S 'm not sure I know a cycling group though ...LoL  @huffeec I can offer a bit if help on a un conference if you like ?
